### ISSUE TITLE
paddle如何实现以下操作

### ISSUE THREAD

**Issue by XueAdas** [PaddlePaddle/Paddle#60376]
### 问题描述 Please describe your issue

                    mask_index = [0, 2]
                    mask = torch.ones([3, 3])
                    img_mask = torch.Tensor([2,2,2])
                    print(mask)
                    mask[:, mask_index] = mask[:, mask_index] * img_mask.unsqueeze(-1)
                    print(mask)

**Comment by zoooo0820** [comment]
你好，目前这个的确是一个Bug。我们会尽快修复。

短期绕过的话 ，右值`mask[:, mask_index] * img_mask.unsqueeze(-1)` 可以再额外加一个`transpose((1,0))`的操作

**Comment by XueAdas** [comment]
您发的邮件已收到，谢谢！
 
Your email has been received, thank you!
Ihre e - mail bekommen, danke!
あなたのメールが届きましたが、ありがとうございます！——————————————————————————Xue Xu    Tel: ***@***.***
